src/Countries/US/Missouri/StLouisEmployer/V20200101/StLouisEmployer.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Missouri\StLouisEmployer\V20200101;
+
+use Appleton\Taxes\Countries\US\Missouri\StLouisEmployer\StLouisEmployer as BaseStLouisEmployer;
+use Illuminate\Database\Eloquent\Collection;
+
+class StLouisEmployer extends BaseStLouisEmployer
+{
+    const TAX_RATE = 0.0005;
```

Comment:
isn't this .05%?

Reply:
Oops good catch, apparently I need to go back to middle school

Reply:
I had to think about it and double check myself

Reply:
I literally googled it to make sure. I was looking at them thinking "No, they're right"